### PR TITLE
Separation of index content

### PR DESCRIPTION
**Pull request by timea-solid**
As discussed in out team meeting and prompted by the https://github.com/solid/webid-profile/issues/35 issue, this is a start to separate indexes.

### CODE REVIEW

**Review by jeff-zucker — CHANGES_REQUESTED** [review]
Removing the Type Index Section and the TOC to it looks good.  Please do NOT remove references to the Type Indexes in other sections (as you do in the storage section).  We need to have a discussion about how this spec will refer to the type index spec but if I'm understanding correctly, @RubenVerborgh  seems to agree with me that we should discuss discovery of type index docs and very generally why we would want to use them in this spec and point to the other spec for all details.

File: solidIndex/index.html [hunk 1]
```diff
@@ -0,0 +1,1603 @@
+<!DOCTYPE html>
```
Comment (RubenVerborgh) [inline solidIndex/index.html:1]: I think there might be a better filename than `solidIndex`.

Shall we call this document `type-index` or `type-indexes`?

**Review by RubenVerborgh — COMMENTED** [review]
I think this is almost good to go.

So @jeff-zucker and I slightly differ:

> @RubenVerborgh seems to agree with me that we should discuss discovery of type index docs

There should indeed be a description somewhere on how to discover type indexes.

However, we might still want to discuss _where_ that description will be written down. Essentially, the question is whether we want:
- the Profile document to have a dependency on the Type Indexes document
- the Type Indexes document to have a dependency on the Profile document

I think the second option is the more sustainable one; indexes will still evolve, but the profile should remain constant.

 Furthermore, the `publicTypeIndex` and `privateTypeIndex` will be defined in the Type Index document; hence, it makes most sense that we also define there how to use them. So that means Type Index will point to Profile in any case; and circular dependencies are the last thing we want.

> very generally why we would want to use them in this spec and point to the other spec for all details.

There might be a generic remark in the Profile document about how it can refer to indexes (in general). Then again, given that a Profile document can refer to anything, that might not be very meaningful.

However, I would refrain from making Profile point to a specific index implementation.

So in that sense, I disagree with:

> Please do NOT remove references to the Type Indexes in other sections (as you do in the storage section).

File: solidIndex/index.html [hunk 2]
```diff
@@ -0,0 +1,1603 @@
...
+            width="72"
+            alt="Solid Project"
+            src="https://solidproject.org/TR/solid.svg"
+        /></a>
+      </address>
+    </header>
+
+    <main>
+      <article about="" typeof="schema:Article doap:Specification">
+        <h1 property="schema:name">Solid WebID Profile</h1>
```
[the remark this answers is not in the document]
Comment (jeff-zucker) [reply solidIndex/index.html:205]: This all makes sense to me.  I withdraw my objection to removing mention of the type indexes in the storage section.  So the policy will be complete separation.  Basically the only time we will mention the type indexes is in the "other predicates" section in which we will basically say that this spec defines what the overall shape of a profile is and what key elements it must contain to be a basic profile but that other specs may introduce additional predicates and requirements.  This actually simplifies the task of the profile spec considerably.

File: solidIndex/index.html (hunk 1 above)
Comment (timea-solid) [reply solidIndex/index.html:1]: At the last meeting we agreed to move this whole part in its own repo. A repo we can name - PROPOSAL: Solid Indexes - and type index will be the specific one described for now in the first version.

File: index.html [hunk 3]
```diff
@@ -1652,8 +1377,7 @@ <h2 property="schema:name">Storage</h2>
                 a WebID owner stores their data. An app wanting to access the
                 WebID owner's Pod should find its location using the
                 <code>pim:storage</code> predicate. Apps looking to access
-                particular types of data should look for specific locations in
-                the typeIndex documents rather than looking for the
+                particular types of data should look for specific locations defined by the index in place rather than looking for the
```
[the remark this answers is not in the document]
Comment (timea-solid) [reply index.html:1380]: For changes that need to affect the webID spec (index.html) we will have a dedicated ticket for it. Jeff agreed to take action on that, given we all are in agreement with Jeff's last comment [down the page](https://github.com/solid/webid-profile/pull/43#discussion_r951542361).

**Review by jeff-zucker — APPROVED** [review]
withdrawing previous objection to removing from storage section

**Comment by timea-solid** [comment]
A repository was set up here: https://github.com/solid/type-indexes and type index-related resources have been moved. This ticket can be closed when this repository is cleaned up too.

File: solidIndex/index.html (hunk 1 above)
Comment (timea-solid) [reply solidIndex/index.html:1]: Decision was done in [last meeting](https://github.com/solid/webid-profile/blob/main/meetings/2022-08-30.md#separation-of-index-content) to have the repo named type-indexes.

**Pull request merged**